src/components/Person/Person.jsx

```diff
@@ -1 +1,19 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const personPartnerParagraph = person.isMarried ? (
+    <p className="Person__partner">
+      {`${person.partnerName} is my ${person.sex === 'm' ? 'wife' : 'husband'}`}
+    </p>
+  ) : (
+    <p className="Person__partner">I am not married</p>
+  );
```

Comment:
Try to avoid nested ternary, instead you can create a separate variable partner and move this part person.sex === 'm' ? 'wife' : 'husband' to it

Reply:
Done